sbc_compassion/models/correspondence_metadata.py

```diff
@@ -42,6 +42,7 @@ def get_correspondence_metadata(self):
         """
         self.ensure_one()
         vals = self.read(self.get_fields())[0]
-        vals['template_id'] = vals['template_id'][0]
+        if vals.get('template_id'):

```

Comment:
Please refine the test to this:
`if isinstance(vals.get('template_id'), tuple):`